app/components/form-security/template.hbs

```diff
@@ -24,7 +24,7 @@
   </div>
   <div class="col-sm-12 col-md-3">
     <div class="input-group">
-      {{input type="number" min="1" step="1" value=memoryMb classNames="form-control" placeholder="Unlimited"}}
+      {{input type="number" min="1" step="1" value=memoryMb classNames="form-control" placeholder=(t 'formSecurity.placeholder.unlimited')}}
```

Comment:
I've been trying to make the lang file consistent so that everything with form fields is in the format of (omitting any parts that aren't needed):

```yaml
componentName:
    fieldName: // usually the variable the 'value' is bound to, `memoryMb` in this case
        label: the label
        placeholder: the placeholder
        unit: MB
        maybeOtherStuff: more things
```